custom/icds_reports/tests/fixtures/child_cases.csv

```diff
@@ -1299,8 +1299,8 @@ ff7feb43-b8d7-4ad0-939c-553e36acf8b0,2017-07-01,Name 2381,2017-06-28,2017-04-19,
 ff97a83f-a9d5-428c-86fe-8c195230a55d,2017-07-01,Name 2738,2017-06-19,2016-12-19,,0,d5d0fce5e73ff2b04417f40bd2bddf4a,ff97a83f-a9d5-428c-86fe-8c195230a55d,ce1e4cca-1530-4899-af49-a556d579c323,10c8b657-370d-4d1a-8c71-40bfcac81b9d,a9,s1,b1,d1,st1,2011-12-20,M,green,1,,0,0,2017-02-13,1,0,0,0,0,0,0,0,0,0,1,0,,,0,yes,no,sc,no,Name 2738,भंवर बाई,,,,0,1,
 ffbc29af-57f5-4e7a-87f2-720c724a649a,2017-07-01,Name 1412,2017-06-28,2017-04-07,,0,d5d0fce5e73ff2b04417f40bd2b8710c,ffbc29af-57f5-4e7a-87f2-720c724a649a,e751978e-aec6-43d4-816d-a8a29f4b0516,37b2c6e3-2903-4267-b393-3fd7967c4481,a20,s4,b2,d1,st1,2012-09-17,M,green,1,,0,0,2017-06-15,1,0,0,0,0,0,0,0,0,0,1,0,,,0,yes,no,obc,no,Name 1412,अनिता,,,,0,1,
 fff0750a-3519-415f-915a-0f5233c81c73,2017-07-01,Name 1508,2017-06-23,2017-05-02,,0,d5d0fce5e73ff2b04417f40bd2b7b165,fff0750a-3519-415f-915a-0f5233c81c73,ab2ab93e-d927-421f-9d39-b8183ab96aac,1d05300f-ff4a-4151-adae-f117e84dfdbf,a24,s8,b4,d3,st2,2013-05-02,F,,1,,0,0,,0,0,0,0,0,0,0,0,0,0,0,1,,,0,yes,no,st,no,Name 1508,सिमाबाई,,,,0,1,
-test_child_case_1,2017-07-01,Name 1508,2017-06-23,2017-05-02,,0,d5d0fce5e73ff2b04417f40bd2b7b165,test_child_case_1,ab2ab93e-d927-421f-9d39-b8183ab96aac,1d05300f-ff4a-4151-adae-f117e84dfdbf,a101,s20,b5,d4,st3,2013-05-02,F,,1,,0,0,,0,0,0,0,0,0,0,0,0,0,0,1,,,0,yes,no,st,no,Name 1508,सिमाबाई,,,,0,1,
-test_child_case_2,2017-07-01,Name 1508,2017-06-23,2017-05-02,,0,d5d0fce5e73ff2b04417f40bd2b7b165,test_child_case_2,ab2ab93e-d927-421f-9d39-b8183ab96aac,1d05300f-ff4a-4151-adae-f117e84dfdbf,a102,s21,b6,d5,st4,2013-05-02,F,,1,,0,0,,0,0,0,0,0,0,0,0,0,0,0,1,,,0,yes,no,st,no,Name 1508,सिमाबाई,,,,0,1,
-test_child_case_3,2017-07-01,Name 1508,2017-06-23,2017-05-02,,0,d5d0fce5e73ff2b04417f40bd2b7b165,test_child_case_3,ab2ab93e-d927-421f-9d39-b8183ab96aac,1d05300f-ff4a-4151-adae-f117e84dfdbf,a103,s22,b7,d6,st5,2013-05-02,F,,1,,0,0,,0,0,0,0,0,0,0,0,0,0,0,1,,,0,yes,no,st,no,Name 1508,सिमाबाई,,,,0,1,
-test_child_case_4,2017-07-01,Name 1508,2017-06-23,2017-05-02,,0,d5d0fce5e73ff2b04417f40bd2b7b165,test_child_case_4,ab2ab93e-d927-421f-9d39-b8183ab96aac,1d05300f-ff4a-4151-adae-f117e84dfdbf,a104,s23,b8,d7,st6,2013-05-02,F,,1,,0,0,,0,0,0,0,0,0,0,0,0,0,0,1,,,0,yes,no,st,no,Name 1508,सिमाबाई,,,,0,1,
-test_child_case_5,2017-07-01,Name 1508,2017-06-23,2017-05-02,,0,d5d0fce5e73ff2b04417f40bd2b7b165,test_child_case_5,ab2ab93e-d927-421f-9d39-b8183ab96aac,1d05300f-ff4a-4151-adae-f117e84dfdbf,a105,s24,b9,d8,st7,2013-05-02,F,,1,,0,0,,0,0,0,0,0,0,0,0,0,0,0,1,,,0,yes,no,st,no,Name 1508,सिमाबाई,,,,0,1,
+test_child_case_1,2017-07-01,Name 1508,2017-06-23,2017-05-02,,0,d5d0fce5e73ff2b04417f40bd2b7b165,test_child_case_1,ab2ab93e-d927-421f-9d39-b8183ab96aac,new_person_case1,a101,s20,b5,d4,st3,2013-05-02,F,,1,,0,0,,0,0,0,0,0,0,0,0,0,0,0,1,,,0,yes,no,st,no,Name 1508,सिमाबाई,,,,0,1,
```

Comment:
By "correct the fixture", did you mean that these child_health cases were originally using the same person case (which is an invalid state)?

In the future can you add that information in a github comment, or the commit body?

Reply:
hey Yes, Initially these were pointing to same person case which is not a valid state.
Yes, Sure I will be putting this explanation on the commit  from next time.